server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -2121,7 +1637,17 @@ public AsyncJobStatusDTO importManifestAsync(
             JobConfig config =  manifestManager.importManifestAsync(owner, fileData.getData(),
                 fileData.getUploadedFilename(), overrides);
 
-            AsyncJobStatus job = this.jobManager.queueJob(config);
+            AsyncJobStatus job;
```

Comment:
Same questions here

Reply:
The auto-generated method does not throws an exception.